ui/app/router.js

```diff
@@ -15,6 +15,7 @@ Router.map(function() {
       this.route('logout');
       this.mount('open-api-explorer', { path: '/api-explorer' });
       this.route('license');
+      this.route('meep'); // NOTE: add query params here, ex: { path: '/:param' }
```

Comment:
the colon indicates it's optional. https://guides.emberjs.com/v3.8.0/routing/query-params/

Reply:
Another note.  We are using Ember version 3.8.  The most recent docs will always land on version 3.22, which is quite different in structure to how our code looks.  Make sure on the upper-left side of the docs pages to select Ember version 3.8 when looking at the documentation.